### ISSUE TITLE
seg fault on connect

### ISSUE THREAD

**Issue by x-sven** [radarku/ardupilot-sitl-docker#20]
Hi, thanks for the nice project. I've build the container myself. It worked and stated waitong for a connection. When I connect either by MissionPlanner or QGCS the container terminates with the following error log.

Any suggestions how to debug/fix it? It may be an issue of the ardupilot code used in the container. Please suggest a stable version if possible!?

Best, Sven

---
Set parameter SIM_SPEEDUP to 1.000000
Home: 42.389800 -71.147600 alt=14.000000m hdg=270.000000
Starting sketch 'ArduCopter'
Starting SITL input
Using Irlock at port : 9005
bind port 5760 for 0
Serial port 0 on TCP port 5760
Waiting for connection ....
Connection on serial port 0
Loaded defaults from /ardupilot/Tools/autotest/default_params/copter.parm
Smoothing reset at 0.001
bind port 5762 for 2
Serial port 2 on TCP port 5762
bind port 5763 for 3
Serial port 3 on TCP port 5763
validate_structures:411: Validating structures
Loaded defaults from /ardupilot/Tools/autotest/default_params/copter.parm
stack overflow in thread FTP

Running: sh Tools/scripts/dumpstack 33 >segv_arducopter.33.out 2>&1
Stack dumped
Tools/scripts/dumpstack: 14: Tools/scripts/dumpstack: gdb: not found

**Comment by uxduck** [comment]
Are you on windows arm64 @x-sven?

**Comment by x-sven** [comment]
no, my docker runns in alpine on a amd64 plattform. i'm getting a connection first and than the container terminates in the connection process, most liekely on parameter download

**Comment by TacticalLaptopBag** [comment]
I'm getting the same error, any fix?
